src/build/do_fetch_litecore.ps1

```diff
@@ -13,14 +13,15 @@ if($DebugLib) {
 
 Write-Host "Fetching variants for $Sha..."
 if($Variants[0].ToLower() -eq "all") {
-    $Variants = @("macosx", "linux", "ios", "android-x86", "android-armeabi-v7a", "android-arm64-v8a", "windows-win32", "windows-win64", "windows-arm")
+    $Variants = @("macosx", "linux", "ios", "android-x86", "android-x86_64", "android-armeabi-v7a", "android-arm64-v8a", "windows-win32", "windows-win64", "windows-arm")
```

Comment:
None of this stuff is relevant to this PR.

Reply:
Not sure why it's appeared here...